spec/config_spec.lua

```diff
@@ -53,38 +62,61 @@ describe("LuaRocks config tests #blackbox #b_config", function()
    end)
 
    describe("LuaRocks config - more complex tests", function()
+      local scdir = testing_paths.testing_lrprefix .. "/etc/luarocks"
+      local versioned_scname = scdir .. "/config-" .. env_variables.LUA_VERSION .. ".lua"
+      local scname = scdir .. "/config.lua"
+
+      local configfile
```

Comment:
Is it? Tests below (lines 77-79, 87-98) use configfile, which is either scname or versioned_scname, which on their turn use scdir.